.github/workflows/test-os.yml

```diff
@@ -194,46 +193,38 @@ jobs:
         name: Cache Vagrant boxes
         uses: actions/cache@v4
         with:
-          path: ~/.vagrant.d/boxes
+          path: /root/.vagrant.d
           key: ${{ runner.os }}-vagrant-${{ hashFiles('hack/Vagrantfile.freebsd13') }}
           restore-keys: |
             ${{ runner.os }}-vagrant-
       -
-        name: Install vagrant and VirtualBox
-        run: |
-          set -x
-          brew tap hashicorp/tap
-          brew install hashicorp/tap/hashicorp-vagrant
-          brew install --cask virtualbox
-      -
-        name: Check versions
+        name: Install vagrant
         run: |
           set -x
+          sudo apt-get update
+          sudo apt-get install -y libvirt-daemon libvirt-daemon-system vagrant vagrant-libvirt
+          sudo systemctl enable --now libvirtd
           vagrant --version
-          VBoxManage -v
       -
         name: Set up vagrant
         run: |
-          vagrant up --no-tty
+          ln -sf hack/Vagrantfile.freebsd13 Vagrantfile
```

Comment:
needs symlink as `VAGRANT_VAGRANTFILE` doesn't seem to be taken into account anymore